pkg/providers/cloudstack/validator.go

```diff
@@ -45,7 +45,9 @@ type localAvailabilityZone struct {
 	DomainId string
 }
 
+// ProviderCmkClient defines the methods used by Cmk as a separate interface to be mockable when injected into other objects.
 type ProviderCmkClient interface {
+	Close(ctx context.Context) error
```

Comment:
what is this needed for?

Reply:
This was added when I was working on mocking the Cmk client. Will remove.